CYBRHardeningCheck/Main.ps1

```diff
@@ -15,6 +15,7 @@
 [CmdletBinding()]
 param
 (
+    [switch]$ReturnReportPath
```

Comment:
Maybe the switch should be "DisableAutoReportOpening"?
The logic is either return the path (which we can always do, it is harmless and probably good to add to the log as well
or open the report
so if I narrow it down, it is either opening the report or not
Do you agree?

Reply:
@AssafMiron if the report filename has the time stamp. It's always good to get it instead of guessing that.
About the switch DisableAutoReportOpening, in general, I have no problem